ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml

```diff
@@ -1,4 +1,8 @@
 base_images:
+  buildah:
+    name: stable
+    namespace: buildah
+    tag: v1.35
```

Comment:
I don't think we need to create a new image here, as part of the ci process the operator is built and deployed already so it should be available to run the tests against.

Unless I'm missing something?